binary_trees.h

```diff
@@ -0,0 +1,21 @@
+#ifndef _BINART_TREES_H_
+#define _BINART_TREES_H_
```

Comment:
shouldn't be `BINARY_TREE_H` ?

Reply:
I got it from the header of `binary_tree_print`

Reply:
alright